govcd/lb.go

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"fmt"
+	"path"
+	"strings"
+)
+
+func extractNSXObjectIDfromPath(locationPath string) (string, error) {
```

Comment:
Some comments on what this function does would be helpful.
Specifically, why do we expect a 2-element list from SplitPath

Reply:
Added